lib/errors.js

```diff
@@ -24,16 +24,16 @@ class HttpErrorBase extends Error {
   constructor (method, res, body, spec) {
     super()
     this.name = this.constructor.name
-    this.headers = res.headers.raw()
+    this.headers = typeof res.headers.raw === 'function' ? res.headers.raw() : res.headers
```

Comment:
What caused this change?

Reply:
https://github.com/npm/npm-profile/blob/lk/abort-signal/lib/index.js#L73